packages/generate-arg-types/src/index.ts

```diff
@@ -1,9 +1,15 @@
+#!/usr/bin/env -S node --experimental-specifier-resolution=node
```

Comment:
Do we need a separate package for this? Maybe generate only inside sdk?

Reply:
and how to generate in image